src/screens/ParadeMapScreen/Map.test.js

```diff
@@ -172,3 +172,40 @@ describe("moveToCurrentLocation", () => {
     });
   });
 });
+
+describe("handleMarkerPress", () => {
+  it("updates state with stage marker details", () => {
```

Comment:
I think there may be a test earlier in the file that is the same as this. That test could be removed.

Reply:
Line 105, test "updates tileDetails and activeMarker state when stage is clicked" is the one that should be bale to be removed.